packages/docs/src/pages/en/features/icon-fonts.md

```diff
@@ -152,7 +152,7 @@ import { aliases, mdi } from 'vuetify/lib/iconsets/mdi-svg'
 
 export default createVuetify({
   icons: {
-    iconfont: 'mdi',
+    defaultSet: 'mdi',
```

Comment:
Is it bad practice to simply use **default**? We use this convention in `lang` with the **defaultLocale** option, but I'm curious as to your thoughts.

Reply:
I generally avoid `default` in any scenario because it is a reserved word.